packages/design-system/src/Tooltip/styles.module.css

```diff
@@ -0,0 +1,8 @@
+@import url("../themes/default/index.css");
+
+:global(#tooltip-root) {
+  position: absolute;
+  top: 0;
+  width: 100%;
+  z-index: var(--ds-z-index-max);
```

Comment:
should we name it as `--ads-z-index-max`? because later on wds's CSS variable would come in too.